### DISCUSSION TITLE
Created for @Cyanidexd. Which function that executes in Open-Shell when you search for something?

### DISCUSSION THREAD

**Discussion by coddec** [Open-Shell/Open-Shell-Menu#1130] [Q&A]
@Cyanidexd
Hi! Me and a friend are trying to modify the search function in OpenShell to use the software Everything (From VoidTools, the SDK of it). We can't seem to find the actual function that executes when you search for something to integrate our own code, so if you could give us a lead on how your code works, it would be really helpful.

**Comment by Cyanidexd** [comment]
Hey, sorry for the extremely late reply but I didn't integrate Everything into the search bar directly, rather I just have a lil button pop up on top of the search bar to search with Everything; Also I only forked the repo, I don't actively work on the development of OpenShell lol.
If I had to look into it, I'd probs say they just call the same Search that Windows uses, so you'd just need to sift through all the code and find the specific part that does it; quite tedious if you ask me, but good luck, and if you ever figure it out I'd actually love for you to let me know
